people_tool/static/people_tool/css/main.css

```diff
@@ -5,6 +5,12 @@
     pointer-events: none;
 }
 
+
+table.person-info-table {
```

Comment:
Just curious, why are we doing this? Could we have a comment as to what this accomplishes?